draft-robert-mimi-delivery-service.md

```diff
@@ -148,9 +155,18 @@ Client           Delivery Service      Queueing Service
 
 ## Authentication
 
-Clients authenticate to the Delivery Service using the signature key from their
+The Delivery Service has two ways of authenticating a request: User-level
+authentication and client-level authentication.
+
+Client-level authentication is used for client-specific group operations. Here,
+clients authenticate to the Delivery Service using the signature key from their
 respective leaf node in the group and a client-specific identifier.
 
+User-level authentication allows clients, for example, to prove that they belong
+to a certain user they add themselves to a group using server-assistance. To
+achieve user-level authentication, clients authenticate using a user-specific
+public key and use the hash of that key as identifier.
```

Comment:
I guess this is more the for the sake of completeness, we don't actually need user keys for anything at the moment.

Reply:
Fair enough. Let's take one thing at a time.